targets/ramips/profiles/usb_mt7620/config

```diff
@@ -2368,41 +3849,77 @@ CONFIG_PACKAGE_wwan=y
 #
 # Boot Loaders
 #
+# CONFIG_PACKAGE_fconfig is not set
+# CONFIG_PACKAGE_uboot-envtools is not set
 
 #
 # Compression
 #
 # CONFIG_PACKAGE_bzip2 is not set
+# CONFIG_PACKAGE_xz-utils is not set
 CONFIG_PACKAGE_zip=y
 
+#
+# Disc
+#
+# CONFIG_PACKAGE_blkdiscard is not set
+CONFIG_PACKAGE_blkid=y
+# CONFIG_PACKAGE_cfdisk is not set
+CONFIG_PACKAGE_fdisk=y
+# CONFIG_PACKAGE_findfs is not set
+# CONFIG_PACKAGE_lsblk is not set
+# CONFIG_PACKAGE_mdadm is not set
+# CONFIG_PACKAGE_partx-utils is not set
+# CONFIG_PACKAGE_sfdisk is not set
+# CONFIG_PACKAGE_wipefs is not set
+
+#
+# Encryption
+#
+# CONFIG_PACKAGE_certtool is not set
+# CONFIG_PACKAGE_gnutls-utils is not set
+# CONFIG_PACKAGE_px5g-mbedtls is not set
+# CONFIG_PACKAGE_px5g-standalone is not set
+
 #
 # Filesystem
 #
 CONFIG_PACKAGE_badblocks=y
+# CONFIG_PACKAGE_chattr is not set
 # CONFIG_PACKAGE_debugfs is not set
 CONFIG_PACKAGE_disktype=y
-CONFIG_PACKAGE_dosfsck=y
-# CONFIG_PACKAGE_dosfslabel is not set
+# CONFIG_PACKAGE_dosfstools is not set
```

Comment:
I thought this one was required by Gargoyle. Not sure.

Reply:
Surely not if this is not in the config file. The config files are generated using menuconfig in the way I explained in the log for commit e62cb58 If it were a dependency, the menuconfig would have added it.

Anyway, do you want me to add it as dependency for gargoyle, or shall we just leave it as is?